charts/budibase/templates/automation-worker-service-deployment.yaml

```diff
@@ -251,6 +252,11 @@ spec:
       {{ end }}
       restartPolicy: Always
       serviceAccountName: ""
-      {{ if .Values.services.automationWorkers.command }}}
```

Comment:
This was in the complete wrong place. I think we got quite lucky that it didn't break anything.